guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -5,13 +5,24 @@
 If the custom code is executed before and/or after the provisioning process, use custom provisioning snippets to avoid recreating cloned templates.
 For more information about configuring custom provisioning snippets, see {ProvisioningDocURL}Creating_Custom_Provisioning_Snippets_provisioning[Creating Custom Provisioning Snippets] in _{ProvisioningDocTitle}_.
 
+* Delete the `/var/lib/pgsql/data-old/` directory:
++
+[options="nowrap" subs="attributes"]
+----
+# rm -r /var/lib/pgsql/data-old/
+----
++
+{ProductName} creates this directory for backup purposes during the upgrade.
```

Comment:
```suggestion
{Project} creates this directory for backup purposes during the upgrade.
```

Reply:
Yikes, my bad, copied over the original suggestion and forgot to update the attribute as per the discussion we had. >_< Apologies.